tensorrt_llm/_torch/pyexecutor/resource_manager.py

```
@@ -1196,9 +1210,68 @@ def get_batch_cache_indices(
                 self.max_attention_window_vec)]
 
         result = self.impl.get_batch_cache_block_ids(request_ids, window_size)
+        tpb = self.tokens_per_block
+        token_layout = []
+        for req_id, req_blocks in zip(request_ids, result):
```

Ruff (0.15.15)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)